tests/multiplexer/test_channel.py

```diff
@@ -149,7 +149,7 @@ async def test_message_transport_never_lock():
     assert channel.healthy
 
 
-async def test_write_throttling(event_loop):
+async def test_write_throttling(event_loop: asyncio.AbstractEventLoop) -> None:
```

Comment:
Side note: Using the builtin fixture `event_loop` is deprecated. We should use `event_loop = asyncio.get_running_loop` instead.

```
tests/multiplexer/test_channel.py:152
  tests/multiplexer/test_channel.py:152: PytestDeprecationWarning: test_write_throttling is asynchronous and explicitly requests the "event_loop" fixture. Asynchronous fixtures and test functions should use "asyncio.get_running_loop()" instead.
    async def test_write_throttling(event_loop: asyncio.AbstractEventLoop) -> None:
```